chore(readme): improve text (#526)
caos/zitadel: README.md

[![Go Report Card](https://goreportcard.com/badge/github.com/caos/zitadel)](https://goreportcard.com/report/github.com/caos/zitadel)
[![codecov](https://codecov.io/gh/caos/zitadel/branch/master/graph/badge.svg)](https://codecov.io/gh/caos/zitadel)

> This project is in a alpha state. The application will continue breaking until version 1.0.0 is released

## What Is It

> `ZITADEL` is a Cloud Native Identity and Access Management solution. All serverside components are written in `Go` and the management interface, called `console`, is written in `Angular`.

Reviewer: should console also be uppercase?

Author: ```suggestion
`ZITADEL` is a Cloud Native Identity and Access Management solution. All serverside components are written in `Go` and the management interface, called `Console`, is written in `Angular`.
```

> `ZITADEL` is a Cloud Native Identity and Access Management solution. All serverside componentes are written in `Go` and the management interface, called `console`, is written in `Angular`.

Reviewer 2: components (not componentes)

> `ZITADEL` is a Cloud Native Identity and Access Management solution. All serverside componentes are written in `Go` and the management interface, called `console`, is written in `Angular`.

Reviewer 2: ```suggestion
`ZITADEL` is a Cloud Native Identity and Access Management solution. All serverside components are written in `Go` and the management interface, called `console`, is written in `Angular`.
```

> We optimized ZITADEL for the usage as `service provider IAM`. By `service provider` we think of companies who build services for e.g SaaS cases. Often these companies would like to use a IAM where they can register there application and grant other people or companies the right to self manage a set of roles within that application.

Reviewer 2: an IAM (not a IAM)

Author: ```suggestion
We optimized ZITADEL for the usage as `service provider IAM`. By `service provider` we think of companies who build services for e.g SaaS cases. Often these companies would like to use an IAM where they can register there application and grant other people or companies the right to self manage a set of roles within that application.
```

> We optimized ZITADEL for the usage as `service provider IAM`. By `service provider` we think of companies who build services for e.g SaaS cases. Often these companies would like to use a IAM where they can register there application and grant other people or companies the right to self manage a set of roles within that application.

Reviewer 2: ```suggestion
We optimized ZITADEL for the usage as `service provider IAM`. By `service provider` we think of companies who build services for e.g SaaS cases. Often these companies would like to use an IAM where they can register there application and grant other people or companies the right to self manage a set of roles within that application.
```

## How Does It Work

We built `ZITADEL` around the idea that the IAM should be easy to deploy and scale. That's why we tried to reduce external systems as much as possible.

> For example, `ZITADEL` is eventsourced but it does not rely on a pub/sub system to function. We instead built all the functionality right into one binary.

Reviewer 2: Instead we built... (not: We instead built...)

Author: ```suggestion
For example, `ZITADEL` is eventsourced but it does not rely on a pub/sub system to function. Instead we built all the functionality right into one binary.
```

> For example, `ZITADEL` is eventsourced but it does not rely on a pub/sub system to function. We instead built all the functionality right into one binary.

Reviewer 2: ```suggestion
For example, `ZITADEL` is eventsourced but it does not rely on a pub/sub system to function. Instead we built all the functionality right into one binary.
```

> `ZITADEL` only needs `kubernetes` for orchestration and `cockroachdb` as storage.

Reviewer 2: names usually uppercase (Kubernetes, CockroachDB)

Author: ```suggestion
`ZITADEL` only needs `Kubernetes` for orchestration and `Cockroachdb` as storage.
```

> `ZITADEL` only needs `kubernetes` for orchestration and `cockroachdb` as storage.

Reviewer 2: ```suggestion
`ZITADEL` only needs `Kubernetes` for orchestration and `CockroachDB` as storage.
```

## Why Another IAM

In the past we already built a closed sourced IAM and tested multiple others. With most of them we had some issues, either technology, feature, pricing or transparency related in nature. For example we find the idea that security related features like `MFA` should not be hidden behind a paywall or a feature price.
One feature that we often missed, was a solid `audit trail` of all IAM resources. Most systems we saw so far either rely on simple log files or use a short retention for this.

## How To Use It

### Use our free tier

> Stay tuned, we will soon publish how you can register yourself a organisation in our cloud offering `zitadel.ch`.

Reviewer 2: an organisation (not a organisation)

Author: ```suggestion
Stay tuned, we will soon publish how you can register yourself an organisation in our cloud offering `zitadel.ch`.
```

> Stay tuned, we will soon publish how you can register yourself a organisation in our cloud offering `zitadel.ch`.

Reviewer 2: ```suggestion
Stay tuned, we will soon publish how you can register yourself an organisation in our cloud offering `zitadel.ch`.
```

Yes we have a free tier!

### Run your own IAM

> Stay tuned, we will publish soon a guide how you can deploy a `hyperconverged` system with our automation tooling called `ORBOS`.

Reviewer: it would be nice to link orbos repo

Author: Makes sense, will do this

> Stay tuned, we will publish soon a guide how you can deploy a `hyperconverged` system with our automation tooling called `ORBOS`.

Reviewer 2: not sure: Stay tuned, we will soon publish a guide.... (placement of soon)

Author: ```suggestion
Stay tuned, we will soon publish a guide how you can deploy a `hyperconverged` system with our automation tooling called `ORBOS`.
```

> Stay tuned, we will publish soon a guide how you can deploy a `hyperconverged` system with our automation tooling called `ORBOS`.

Reviewer 2: ```suggestion
Stay tuned, we will soon publish a guide how you can deploy a `hyperconverged` system with our automation tooling called `ORBOS`.
```

> With `ORBOS` you will be able to run `ZITADEL` on `GCE` or `StaticMachines` within 20min. To achieve this, `ORBOS` will boostrap a `kubernetes` cluster, install the platform components (logging, metrics, ingress, ...), start a secure `cockroach` cluster and run and operate the `ZITDADEL`.

Reviewer 2: 20 minutes (i would not abb.)

Author: ```suggestion
With `ORBOS` you will be able to run `ZITADEL` on `GCE` or `StaticMachines` within 20 minutes. To achieve this, `ORBOS` will boostrap a `kubernetes` cluster, install the platform components (logging, metrics, ingress, ...), start a secure `cockroach` cluster and run and operate the `ZITDADEL`.
```

> With `ORBOS` you will be able to run `ZITADEL` on `GCE` or `StaticMachines` within 20min. To achieve this, `ORBOS` will boostrap a `kubernetes` cluster, install the platform components (logging, metrics, ingress, ...), start a secure `cockroach` cluster and run and operate the `ZITDADEL`.

Reviewer 2: the Zitadel? I'd just write "...and operate `ZITADEL`.

Author: ```suggestion
With `ORBOS` you will be able to run `ZITADEL` on `GCE` or `StaticMachines` within 20min. To achieve this, `ORBOS` will bootstrap a `Kubernetes` cluster, install the platform components (logging, metrics, ingress, ...), start a secure `cockroach` cluster and run and operate `ZITDADEL`.
```

> With `ORBOS` you will be able to run `ZITADEL` on `GCE` or `StaticMachines` within 20min. To achieve this, `ORBOS` will boostrap a `kubernetes` cluster, install the platform components (logging, metrics, ingress, ...), start a secure `cockroach` cluster and run and operate the `ZITDADEL`.

Reviewer 2: ```suggestion
With `ORBOS` you will be able to run `ZITADEL` on `GCE` or `StaticMachines` within 20 minutes. To achieve this, `ORBOS` will boostrap a `kubernetes` cluster, install the platform components (logging, metrics, ingress, ...), start a secure `cockroach` cluster and run and operate `ZITDADEL`.
```
